README.md

```diff
@@ -76,6 +76,14 @@ $swagger = $serializer->deserialize($jsonString, 'Swagger\Annotations\Swagger');
 echo $swagger;
 ```
 
+### Usage from docker
+
+Generate the swagger documentation to a static json file.
+
+```
+docker run -v `pwd`:/app -it tico/swagger-php --help
```

Comment:
This command will explode in your face if there's special characters in the current path (e.g. folder name with a space).

Better quote the path:

    docker run -v "`pwd`":/app -it tico/swagger-php --help

Even better use the variable that's already populated instead of invoking another program.

    docker run -v "$PWD":/app -it tico/swagger-php --help

Reply:
:+1:  I will change it today